frontend/javascripts/admin/voxelytics/statistics_tab.tsx

```diff
@@ -85,12 +90,27 @@ export default function StatisticsTab({
           {statisticsResult.value.map((row: VoxelyticsChunkStatistics) => (
             <tr key={row.executionId}>
               <td>
-                {row.executionId}
-                <br />
-                <span className="stats-label">
-                  {row.countFinished !== row.countTotal && <>{row.countFinished} of </>}
-                  {row.countTotal} chunk{row.countTotal !== 1 && "s"} completed
-                </span>
+                <Tooltip
+                  overlay={
+                    <>
+                      {row.counts.complete}/{row.counts.total - row.counts.skipped} complete,{" "}
```

Comment:
not sure whether it is intuitive to subtract the skipped ones from `total`. how about: `r remaining, x complete, y cancelled, z failed, s skipped, t total` where `r = total - complete - skipped`.